include/deal.II/physics/physics.h

```diff
@@ -0,0 +1,118 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__physics_h
+#define dealii__physics_h
+
+// This file only provides definition and documentation of the
+// namespace Physics. There is no necessity to include it
+// anywhere in a C++ code. Only doxygen will make use of it.
+
+#include <deal.II/base/config.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+
+/**
+ * A collection of namespaces and utilities to assist in the
+ * definition, construction and manipulation of data related to
+ * physical fields and materials.
+ *
+ * @ingroup Physics
+ */
+namespace Physics
+{
+
+  /**
+   * This namespace provides a collection of definitions that
+   * conform to standard notation used in (nonlinear) elasticity.
+   *
+   * <h3>Notation</h3>
+   *
+   * References for this notation include:
+   *    - G.A. Holzapfel (2001),
```

Comment:
you can put it in a bibtex form (I don't recall exactly the syntax, for sure used in FE_Enriched)